contrib/wl-clipboard/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "wl-clipboard"
+pkgver = "2.2.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+]
+makedepends = ["wayland-devel", "wayland-progs", "wayland-protocols"]
```

Comment:
Perhaps the build instructions aren't quite clear on this (something for me to improve!), but wl-clipboard needs the `wayland-scanner` binary on the machine it's being built _on_, not the machine it's being built _for_ (these are called the _build_ and _host_ machines respectively in GNU and Meson terminology) — `wayland-scanner` is a code generator, it runs at build time. So if I'm reading this right, you should move `wayland-progs` up to `hostmakedepends` (and perhaps try actually cross-compiling it 🙂).

Reply:
cbuild does it backwards- makedepends is target-sysroot when cross, hostmakedepends is current machine when cross

Reply:
(so yes, it should be moved up, but by autoconf terminology the hostdeps would be the wrong place)

Reply:
(Yes, I understood that.)

Reply:
> Perhaps the build instructions aren't quite clear on this (something for me to improve!), but wl-clipboard needs the `wayland-scanner` binary

No that's just my inexperience and unfamiliarity with with the dependencies.

I've updated the template and tested that it cross-compiles.